app/(gcforms)/[locale]/(form administration)/form-builder/[id]/responses/[statusFilter]/components/ManageFormAccessDialog/InviteUsers.tsx

```diff
@@ -0,0 +1,84 @@
+import { useContext, useState } from "react";
+import { ManageFormAccessDialogContext } from "./ManageFormAccessDialogContext";
+import { sendInvitation } from "./actions";
+import { Button } from "@clientComponents/globals";
+import { useTranslation } from "@i18n/client";
+import { toast } from "@formBuilder/components/shared";
+
+export const InviteUsers = () => {
+  const { t } = useTranslation("manage-form-access");
+  const [message, setMessage] = useState("");
+  const [errors, setErrors] = useState<string[]>([]);
+
+  const manageFormAccessDialogContext = useContext(ManageFormAccessDialogContext);
```

Comment:
We should do this outside of the component like we do for the template store etc...

```
export const useTemplateStore = <T,>(
  selector: (state: TemplateStoreState) => T,
  equalityFn?: (left: T, right: T) => boolean
): T => {
  const store = useContext(TemplateStoreContext);
  if (!store) throw new Error("Missing Template Store Provider in tree");
  return useStoreWithEqualityFn(store, selector, equalityFn ?? shallow);
};
```